src/main/java/de/tum/in/www1/artemis/repository/ExamRepository.java

```diff
@@ -16,5 +16,5 @@
 public interface ExamRepository extends JpaRepository<Exam, Long> {
 
     @Query("SELECT e FROM Exam e WHERE e.course.id = :#{#courseId}")
```

Comment:
We don't need the @query here, as the method name should suffice.
See https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.query-methods